codice/ddf
# DDF-2133: Add configurable port number for FTP Endpoint

#### What does this PR do?
Adds a configuration option to the Admin UI to change the port for the FTP Endpoint
#### Who is reviewing it (please choose AT LEAST two reviewers that need to approve the PR before it can get merged; if a component team is listed, at least one of its members needs to approve)?
@troymohl @mweser @ricklarsen @emmberk
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).
@jaymcnallie
@kcwire
#### How should this be tested?
-Install the FTP Endpoint feature under the DDF Catalog application.
-FTP into FTP server using default port of **8021**.
-In the Admin UI, change the FTP port to any other unprivileged port that is not in use.
-Verify you can FTP into server using new port.
#### Any background context you want to provide?
#### What are the relevant tickets?
DDF-2133
#### Screenshots (if appropriate)
#### Checklist:
